dist/main/index.js

```diff
@@ -139182,6 +139185,19 @@ class GradleStateCache {
         const absolutePath = path_1.default.resolve(__dirname, '..', '..', 'sources', 'src', 'resources', ...paths);
         return fs_1.default.readFileSync(absolutePath, 'utf8');
     }
+    configureInfoLogLevel() {
+        const infoProperties = `org.gradle.logging.level=info\norg.gradle.logging.stacktrace=all\n`;
```

Comment:
I used `\n` here, there is also `os.EOL` but I have no experience with TypeScript/nodeJS.